tests/test_text.py

```diff
@@ -0,0 +1,775 @@
+# -*- coding: utf-8 -*-
+# Copyright FMR LLC <opensource@fidelity.com>
+# SPDX-License-Identifier: GNU GPLv3
+import numpy as np
+import pandas as pd
+import random
+import sklearn
+from feature.selector import Selective, SelectionMethod
+from tests.test_base import BaseTest
+from textwiser import TextWiser, Embedding, Transformation
+
+
+class TestText(BaseTest):
+
+    ################################################
+    ########## Tests for random selection ##########
+    ################################################
+
+    # Check Random with unicost and diverse (the same features should select)
+    # Check solution for large trails with the same seed (the same features should select)
+    def test_text_based_random_num_feature(self):
```

Comment:
The only other test I would recommend is "test_usage_example" which can be the first test in this file that is identical to the usage example to ensure that our usage example works. Add that tests, see if the example works :D and tweak if necessary

Reply:
test_usage_example is added to the test_text.py. Also I edited README.md accordingly.